src/KernelIntegrator/FlatConnectivity.cc

```diff
@@ -20,6 +20,22 @@
 #include "Utilities/DBC.hh"
 #include "Utilities/globalNodeIDs.hh"
 
+namespace { // anonymous
```

Comment:
This method basically repeats what you can get from ConnectivityMap::numNeighborsForNode, and to some extent the DataBase method DataBase::numNeighbors (which computes a FieldList with the result for all points).  Wouldn't the method already in ConnectivityMap be good enough here though?